ref_ts/models.py

```diff
@@ -49,304 +31,50 @@ def can_have_multiple_files(cls):
-                raise ValidationError("The only Variable of the resource can't be deleted.")
-            site.delete()
-        else:
-            raise ObjectDoesNotExist("No Site element was found for id:%d." % element_id)
-
-
-# class Source(AbstractMetaDataElement):
-#     term = 'Source'
-#     derived_from = models.CharField(max_length=300)
-#     organization = models.CharField(max_length=200, null=True)
-#     source_description = models.CharField(max_length=400, null=True)
-#
-#     def __unicode__(self):
-#         return self.derived_from
-#
-#     @classmethod
-#     def create(cls, **kwargs):
-#         if 'derived_from' in kwargs:
-#             # check the source doesn't already exists - source needs to be unique per resource
-#             if 'metadata_obj' in kwargs:
-#                 metadata_obj = kwargs['metadata_obj']
-#                 metadata_type = ContentType.objects.get_for_model(metadata_obj, relate_name='+')
-#                 src = Source.objects.filter(derived_from= kwargs['derived_from'], object_id=metadata_obj.id, content_type=metadata_type,related_name='+').first()
-#                 if src:
-#                     raise ValidationError('Source:%s already exists for this resource.' % kwargs['derived_from'])
-#                 src = Source.objects.create(derived_from=kwargs['derived_from'], organization=kwargs.get('organization'),source_description=kwargs.get('source_description'), content_object=metadata_obj)
-#                 return src
-#             else:
-#                 raise ValidationError('Metadata instance for which source element to be created is missing.')
-#         else:
-#             raise ValidationError("Source data is missing.")
-#
-#     @classmethod
-#     def update(cls, element_id, **kwargs):
-#         src = Source.objects.get(id=element_id)
-#         if src:
-#             if 'derived_from' in kwargs:
-#                 src.derived_from = kwargs['derived_from']
-#             if 'organization' in kwargs:
-#                 src.organization = kwargs['organization']
-#             if 'source_description' in kwargs:
-#                 src.source_description = kwargs['source_description']
-#             src.save()
-#         else:
-#             raise ObjectDoesNotExist("No source element was found for the provided id:%s" % element_id)
-#
-#     @classmethod
-#     def remove(cls, element_id):
-#         src = Source.objects.get(id=element_id)
-#         if src:
-#             src.delete()
-#         else:
-#             raise ObjectDoesNotExist("No source element was found for id:%d." % element_id)
+    name = models.CharField(max_length=500, default="", blank=True)
+    code = models.CharField(max_length=500, default="", blank=True)
+    net_work = models.CharField(max_length=500, default="", blank=True)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, blank=True)
```

Comment:
The regular time series resource uses the Dublin Core coverage element to express the coordinates of the monitoring site.